.ci/aui/buildContainerImage.js

```diff
@@ -8,4 +8,4 @@ const tag = getRequiredEnvVar("TAG");
 const platforms = process.env.PLATFORMS ?? "linux/amd64,linux/arm64";
 const push = process.env.PUSH === "1" ? ["--push", "--provenance=true", "--sbom=true"] : "";
 
-await $`docker buildx build --file ./AdminApi/src/AdminApi/Dockerfile --tag ghcr.io/nmshd/backbone-admin-ui:${tag} --platform ${platforms} --build-arg VERSION=${tag} ${push} .`;
+await $`docker buildx build --file .Executables/AdminApi/src/AdminApi/Dockerfile --tag ghcr.io/nmshd/backbone-admin-ui:${tag} --platform ${platforms} --build-arg VERSION=${tag} ${push} .`;
```

Comment:
```suggestion
await $`docker buildx build --file ./Executables/AdminApi/src/AdminApi/Dockerfile --tag ghcr.io/nmshd/backbone-admin-ui:${tag} --platform ${platforms} --build-arg VERSION=${tag} ${push} .`;
```

Reply:
Same for some of the other `buildDockerImage` files